snowflake/tests/snowflake_connector_patch/_snowflake_connector_patch/tables.py

```diff
@@ -45,3 +45,79 @@
         Decimal('0.000000'),
     )
 ]
+ORGANIZATION_USAGE_IN_CURRENCY_DAILY = [('test', 'Standard', 'Compute', 'USD', Decimal('0.4'), Decimal('0.7'))]
```

Comment:
Since we are adding these here maybe we can add assertions for these metrics on the e2e test, which is where this data is used?

Reply:
Sure